### PR TITLE
Fix text track support in snapshot

### PR DESCRIPTION
**Pull request by incompl**

### CODE REVIEW

File: src/plugin.js [hunk 1]
```diff
@@ -206,6 +206,9 @@ const contribAdsPlugin = function(options) {
 
   player.ads.adMacroReplacement = adMacroReplacement.bind(player);
 
+  // Expose snapshot tools for testing
+  player.ads._snapshot = snapshot;
```
Comment (ldayananda) [inline src/plugin.js]: Why is this not already available to tests via player.ads.snapshot? Can we import it into the test file instead?

Comment (ldayananda) [inline src/plugin.js]: What I meant was: is it possible to import the snapshot module into the test file?

**Comment by ldayananda** [comment]
LGTM

**Comment by pagarwal123** [comment]
QA-LGTM

**Pull request merged**